doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -119,46 +260,61 @@ documentation](https://helm.sh/docs/intro/install/).
           cidrs: ["0.0.0.0/0"]
         internal:
           enabled: true
+
       ```
 
    For production, if you have [opted for locally-attached storage](/self-managed/operational-guidelines/#locally-attached-nvme-storage-openebs),
-   include the storage configuration your configuration file.  See the [Locally-attached NVMe storage (OpenEBS)](/self-managed/operational-guidelines/#locally-attached-nvme-storage-openebs)
-
-1. Clone/download the [Materialize
-   repo](https://github.com/MaterializeInc/materialize).
-
-1. Go to the Materialize repo directory.
-
-   ```bash
-   cd materialize
-   ```
+   include the storage configuration your configuration file.  See the
+   [Locally-attached NVMe storage
+   (OpenEBS)](/self-managed/operational-guidelines/#locally-attached-nvme-storage-openebs)
+   for details.
 
-1. Install the Materialize operator with the release name
-   `my-materialize-operator`, specifying the path to your `my-AWS-values.yaml`
-   file:
+1. Install the Materialize operator `materialize-operator`, specifying the path
+   to your `my-materialize-operator-values.yaml` file:
 
    ```shell
-   helm install my-materialize-operator -f path/to/my-AWS-values.yaml materialize/misc/helm-charts/operator
+   helm install materialize-operator misc/helm-charts/operator \
+      -f my-materialize-operator-values.yaml  \
+      --namespace materialize --create-namespace
    ```
 
-1. Verify the installation:
+1. Verify the installation and check the status:
 
     ```shell
     kubectl get all -n materialize
     ```
 
-## B. Install Materialize
+    Wait for the components to be in the `Running` state:
+
+    ```none
+    NAME                                        READY   STATUS    RESTARTS   AGE
+    pod/materialize-operator-84ff4b4648-brjhl   1/1     Running   0          12s
+
+    NAME                                   READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/materialize-operator   1/1     1            1           12s
+
+    NAME                                              DESIRED   CURRENT   READY   AGE
+    replicaset.apps/materialize-operator-84ff4b4648   1         1         1       12s
+    ```
+
+    If you run into an error during deployment, refer to the
+    [Troubleshooting](/self-hosted/troubleshooting) guide.
+
+## C. Install Materialize
 
 To deploy Materialize:
 
-1. Create a [Kubernetes
-   Secret](https://kubernetes.io/docs/concepts/configuration/secret/) for your
-   backend configuration information and save in a file (e.g.,
-   `materialize-backend-secret.yaml`).
+<a name="deploy-materialize-secrets"></a>
+
+1. {{< warning>}}
+
+   {{< self-managed/terraform-disclaimer >}}
```

Comment:
hammering home the disclaimer is important, but imo we don't need it a second time here

Reply:
👍  Also, I guess this whole section may go away if we get Terraform to deploy materialize itself.

Reply:
Updated in patch